user-management/src/main/elm/sources/UserManagement.elm

```diff
@@ -105,23 +105,23 @@ update msg model =
         AddUser result ->
              case result of
                  Ok username ->
-                     (model, postReloadConf model )
+                     (model, getUsersConf model )
```

Comment:
we don't need to ask for a reload, it's the role of the backend to ensure backend logic. In fact, the the API return value should be enough to update the UI as wanted.